doc/releases/changelog-dev.md

```diff
@@ -239,17 +243,10 @@
   1: ──RX(0.70)─┤ ╰<Z@Z>
   ```
 
-* The :class:`~.CompilePipeline` (previously known as the `TransformProgram`) can now be constructed
-  more flexibility with a variable number of arguments that are of types :class:`~.transforms.core.Transform`,
-  :class:`~.transforms.core.BoundTransform`, or other `CompilePipeline`s.
+* A :class:`~.CompilePipeline` can be initialized by passing any number of transforms or other ``CompilePipeline``s
+  to the ``transforms`` argument, providing more flexibility than the previous ``TransformProgram`` class.
```

Comment:
This sounds like we're promoting
```python
qml.CompilePipeline(transforms=[...])
```
which is not really the recommended way of doing things. The actual call signature should be
```python
qml.CompilePipeline(transform1, transform2, ...)
```

Reply:
https://github.com/PennyLaneAI/pennylane/pull/8818/commits/9a112565d0ae9b9b94ec32953fa198a6dc858184